runtime/hydradx/src/evm/mod.rs

```diff
@@ -137,53 +137,54 @@ parameter_types! {
 	pub const SuicideQuickClearLimit: u32 = 0;
 }
 
-impl pallet_evm::Config for crate::Runtime {
-	type AddressMapping = ExtendedAddressMapping;
-	type BlockGasLimit = BlockGasLimit;
+impl pallet_evm::Config for Runtime {
```

Comment:
dont reorder type definition pls

Reply:
Ok. I just like to have them in the same order as defined in `Config`.